.github/workflows/pr.yml

```diff
@@ -61,8 +64,12 @@ jobs:
           restore-keys: ${{ runner.os }}-sbt
       - name: Backend tests
         shell: bash
-        #TODO: coverage?
-        run: ./ciRunSbt.sh test
+        run: ./ciRunSbt.sh coverage test coverageReport
+      - name: Upload Coverage data to Coveralls
+        shell: bash
+        run: ./ciRunSbt.sh coverageAggregate coveralls
+        env:
+          COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are you sure that should be `GITHUB_TOKEN`?

Reply:
I just followed: https://github.com/marketplace/actions/coveralls-github-action#inputs
It seems to be working